server/app.js

```diff
@@ -6,7 +6,7 @@ const app = express();
 
 
 app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({extended: false}));
+app.use(bodyParser.urlencoded({extended: true}));
```

Comment:
why did you change the extended `options` to `true` ?

Reply:
Since false only accepts values of a key pair to be string or array, I just thought I should change it to true so that it can accept any type, just in case